app/controllers/sign_in/webhooks/logingov_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SignIn
+  module Webhooks
+    class LogingovController < SignIn::ServiceAccountApplicationController
+      before_action :authenticate_service_account
+
+      def risc
+        head :accepted
+      end
+
+      private
+
+      def authenticate_service_account
+        jwt = request.raw_post
+
+        @risc_jwt = SignIn::Logingov::Service.new.jwt_decode(jwt)
+      rescue SignIn::Logingov::Errors::JWTDecodeError => e
```

Comment:
I think we could simplify this with having only one rescue:
`rescue Errors::StandardError => e`

And then we can just log the message:
`Rails.logger.error("[SignIn][Webhooks][LogingovController] error", e)`

And render:
`render json: { error: e }, status: :unauthorized`